mlir/lib/Bindings/Python/IRAttributes.cpp

```diff
@@ -1426,7 +1446,7 @@ py::object symbolRefOrFlatSymbolRefAttributeCaster(PyAttribute &pyAttribute) {
 
 void mlir::python::populateIRAttributes(py::module &m) {
   PyAffineMapAttribute::bind(m);
-
+  PyIntegerSetAttribute::bind(m);
```

Comment:
Can we please preserve the alphabetical order?

Reply:
Addressed in the force-push below.  I should have kept it as two commits and squashed at the end.  Please have a look below for this time.  Thanks!